app/routes/user/email_notifications.server.ts

```diff
@@ -0,0 +1,208 @@
+/*!
+ * Copyright © 2022 United States Government as represented by the Administrator
+ * of the National Aeronautics and Space Administration. No copyright is claimed
+ * in the United States under Title 17, U.S. Code. All Other Rights Reserved.
+ *
+ * SPDX-License-Identifier: NASA-1.3
+ */
+import { tables } from '@architect/functions'
+import { mapTopicsToFormatAndNoticeType } from '~/lib/utils'
+import { getUser } from '~/routes/__auth/user.server'
+
+// db model
+export type EmailNotification = {
+  name: string
+  recipient: string
+  created: number
+  active: boolean
+  uuid?: string
+  topics: string[]
+}
+
+// view model
```

Comment:
What do you mean by view model?

```suggestion
// inverse table
```

Reply:
This is probably a less applicable term in the context of what we are going, but the idea is that the View Model is what the frontend works with; the model that the 'view' page is using. 

In this case, the fields like format and notice type are not 'real' in the sense of how the tables are storing data, but are needed as distinct for the UI.